### PR TITLE
prysm alerts notifications 

### PR DESCRIPTION
**Pull request by gbayasgalan**
1. alerts panel is added into multiple-scource.json dashboard 
2. grafana image renderer is configured in docker-compose.yaml

### CODE REVIEW

File: compose-examples/prysm-only/docker-compose.yaml [hunk 1]
```diff
@@ -91,11 +91,27 @@ services:
...
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000
+      GF_LOG_FILTERS: rendering:debug
+    <<: *logging
+
+  renderer:
+    image: grafana/grafana-image-renderer:2.0.0
+    restart: on-failure
+    hostname: renderer
+    ports:
+      - 127.0.0.1:8082:8082
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:111]: don't know if this public port is necessary, grafana is accessing the renderer via docker internal network

File: compose-examples/prysm-only/docker-compose.yaml [hunk 2]
```diff
@@ -91,11 +91,27 @@ services:
...
       - ./config/grafana/provisioning-p:/etc/grafana/provisioning
       - ./data/grafana:/var/lib/grafana
+    environment:
+      GF_RENDERING_SERVER_URL: http://renderer:8082/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000
+      GF_LOG_FILTERS: rendering:debug
+    <<: *logging
+
+  renderer:
+    image: grafana/grafana-image-renderer:2.0.0
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:107]: Can you put this version number into `.env` please like grafana and prometheus?

File: compose-examples/prysm-only/docker-compose.yaml [hunk 3]
```diff
@@ -91,11 +91,27 @@ services:
...
+      - renderer
     ports:
       - 127.0.0.1:3000:3000
     volumes:
       - ./config/grafana/provisioning-p:/etc/grafana/provisioning
       - ./data/grafana:/var/lib/grafana
+    environment:
+      GF_RENDERING_SERVER_URL: http://renderer:8082/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000
+      GF_LOG_FILTERS: rendering:debug
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:103]: debug necessary?

File: compose-examples/prysm-only/docker-compose.yaml [hunk 4]
```diff
@@ -91,11 +91,27 @@ services:
...
+    <<: *logging
+
+  renderer:
+    image: grafana/grafana-image-renderer:2.0.0
+    restart: on-failure
+    hostname: renderer
+    ports:
+      - 127.0.0.1:8082:8082
+    environment:
+      ENABLE_METRICS: 'true'
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:113]: can you also connect this container to prometheus?

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/prysm-only/docker-compose.yaml (hunk 2 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:107]: Tag (2.0.0) is added into .env

File: compose-examples/prysm-only/docker-compose.yaml (hunk 1 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:111]: you right, it was just for testing metrics and it's removed now

File: compose-examples/prysm-only/docker-compose.yaml (hunk 4 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:113]: now render is connected to prometheus

File: compose-examples/prysm-only/docker-compose.yaml (hunk 3 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:103]: not really, I used it for log msg for testing purpose, now it's removed

File: compose-examples/prysm-only/docker-compose.yaml [hunk 5]
```diff
@@ -100,15 +100,14 @@ services:
...
     <<: *logging
 
   renderer:
-    image: grafana/grafana-image-renderer:2.0.0
+    image: grafana/grafana-image-renderer:${GRAFANA_IMAGE_RENDERER_TAG}
     restart: on-failure
     hostname: renderer
     ports:
-      - 127.0.0.1:8082:8082
+      - 8082
```
Comment (stefa2k) [inline compose-examples/prysm-only/docker-compose.yaml:110]: I think this is not necessary at all... is there any service accessing this container from outside docker?

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: compose-examples/prysm-only/docker-compose.yaml (hunk 5 above)
Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:110]: at the moment only "grafana" service accessing it

Comment (stefa2k) [reply compose-examples/prysm-only/docker-compose.yaml:110]: but grafana is inside docker, not from outside of the host. Can you please try remove this port binding and see if it still works?

Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml:110]: yeah sure, just minut

Comment (gbayasgalan) [reply compose-examples/prysm-only/docker-compose.yaml]: it's removed and works fine

**Review by stefa2k — APPROVED** [review]
lgtm

**Pull request merged**